.eslintrc.js

```diff
@@ -65,6 +61,7 @@ module.exports = {
       rules: Object.assign({}, require('eslint-plugin-node').configs.recommended.rules, {
         // add your custom rules and overrides for node files here
         'ember/avoid-leaking-state-in-ember-objects': 'off',
+        'node/no-unpublished-require': 'off'
```

Comment:
Is this just for `electron`? Could we configure it explicitly with `"allowModules": [],` instead of disabling the rule entirely?